src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const VERY_SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCAUNT = 50;
+  const OVER_SHORT_TERM = 6;
+  const PRE_SHORT_TERM = 3;
+  const SHORT_TERM_DISCAUNT = 20;
```

Comment:
There is a typo in the constant name 'SHORT_TERM_DISCAUNT'. It should be 'SHORT_TERM_DISCOUNT'. Correct spelling is crucial for maintaining clear and understandable code.